tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotSerializer.cs

```diff
@@ -191,8 +191,6 @@ public static void SerializeStaticFields(Type declaringType, JsonTextWriter json
 
             if (currentDepth >= _maximumDepthOfMembersToCopy)
             {
-                jsonWriter.WritePropertyName("type");
```

Comment:
Unless the type gets written somewhere else, this removal seems to break the spec described in the internal RFC? (see https://docs.google.com/document/d/10ViZr1Tgj3L8tWBqHw9krPjraYxXvJE-2FZXOys8NVI/edit#bookmark=id.rm9rwlqki587 specifically)

Reply:
The `type` is already written in `WriteTypeAndValue`.

Reply:
Got it, makes sense! Resolving.